package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@oxc-resolver/binding",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "build": "napi build --platform --release --manifest-path napi/Cargo.toml",
+    "build:debug": "napi build --platform --manifest-path napi/Cargo.toml",
+    "prepublishOnly": "napi pre-publish -t npm",
+    "test": "ava"
+  },
+  "devDependencies": {
+    "@emnapi/core": "^0.44.0",
+    "@emnapi/runtime": "^0.44.0",
+    "@napi-rs/cli": "^3.0.0-alpha.22",
+    "@tybys/wasm-util": "^0.8.0",
+    "@types/node": "^20.10.4",
+    "ava": "^6.0.1",
+    "emnapi": "^0.44.0",
+    "typescript": "^5.3.3"
+  },
```

Comment:
Do we need all these?

Reply:
`emnapi` stuff is for compiling `wasm32-wasi` target. `ava` is for testing. `TypeScript` and `@types/node` is for code completion in IDEs